infra/blueprint-test/pkg/utils/env.go

```diff
@@ -31,3 +31,11 @@ func ValFromEnv(t testing.TB, k string) string {
 	}
 	return v
 }
+
+// SetEnv set a environment variable.
+func SetEnv(t testing.TB, key string, value string) {
+	err := os.Setenv(key, value)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Thanks for refactoring! We could also add some context to the error as to which key failed.